<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Marabou</class>
 <widget class="QWidget" name="Marabou">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>679</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Network File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="network_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="importNetwork">
       <property name="text">
        <string>import</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QCheckBox" name="ar_checkbox">
       <property name="text">
        <string>Use Abstract and Refine</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>240</height>
      </size>
     </property>
     <property name="title">
      <string>Abstract</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Abstraction Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="abstract_type_combobox">
          <item>
           <property name="text">
            <string>complete</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>heuristic_alg2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>heuristic_random</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>property :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="property_combobox">
          <item>
           <property name="text">
            <string>property_1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>property_2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>property_3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>property_4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>sanity_UNSAT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>sanity_SAT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>basic_0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>basic_1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>basic_2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_17</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adversarial_19</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>basic_medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>basic_long-medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>basic_long</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>basic_default</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>abstraction_sequence :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="abstraction_sequence_slider">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="as_value">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="run_abstract">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Refine</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="refine_next">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refine_finish">
          <property name="text">
           <string>Finish</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Result:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">background:rgb(0, 170, 0);border-radius:100%;width:100%;height:100%;margin-right:20px;text-align:center;</string>
       </property>
       <property name="text">
        <string>SAT</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="verify_without_ar">
       <property name="text">
        <string>Verify_without_AR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="verify_with_ar">
       <property name="text">
        <string>Verify_with_AR</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
